src/utils/brand.utils.ts

```diff
@@ -0,0 +1,34 @@
+import { brandConfig, landingCompanies, platforms } from "../constants/brand.constants";
+
+/**
+ * This function will check whether the landing company is available in our brand configuration
+ *
+ * @param {landingCompanies} landingCompany - landingCompany will be the string and we will check if its available in legal entities.
+ * @returns {string} Returns name of landing company.
+ */
+export const getLegalEntityName = (landingCompany: keyof typeof landingCompanies): string => {
```

Comment:
```suggestion
export const getLegalLandingCompany = (landingCompany: LandingCompanies) => {
```

Please import `LandingCompanies` type and use it like this